### PR TITLE
Fix hacky next_step handling

### PR DESCRIPTION
**Pull request by melindaloubser1**
- closes https://github.com/RasaHQ/rasa-demo/issues/496
- Use a follow up action instead of a user utterance to trigger `action_greet_user` for `/next_step{"step":null}`

### CODE REVIEW

File: actions/actions.py [hunk 1]
```diff
@@ -643,17 +644,7 @@ def run(self, dispatcher, tracker, domain) -> List[EventType]:
...
-            return [
-                ActionExecuted("action_listen"),
-                UserUttered(
-                    "/get_started_step1",
-                    {
-                        "intent": {"name": "get_started_step1", "confidence": 1.0},
-                        "entities": [],
-                    },
-                ),
-            ]
```
Comment (erohmensing) [inline actions/actions.py:656]: I'm confused about this, does `action_greet_user` usually happen after `get_started_step1`? shouldn't it go to the beginning of the onboarding flow?

Comment (melindaloubser1) [reply actions/actions.py:656]: Yeah, I was confused too, but it looks like this is a kinda rare scenario where the user says "next step", before any steps have been completed, or maybe after all steps have been completed.

Comment (melindaloubser1) [reply actions/actions.py:656]: e.g. start the conversation with "hi", then immediately after write "next step please"

Comment (melindaloubser1) [reply actions/actions.py:656]: There's a lot of these kinds of stories: 
```
## Story from conversation with cfa8bb9deaf0427498c662745431a282 on December 15th 2018
* get_started_step1
    - action_greet_user
    - slot{"shown_privacy":true}
* chitchat
    - respond_chitchat
* enter_data
    - utter_not_sure
    - utter_possibilities
```

**Pull request merged**